.github/workflows/build-and-test.yaml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go-version: ['1.21', '1.22', '1.23']
+        go-version: ['1.14']
```

Comment:
Please file an issue to upgrade the Go version too ?

Reply:
Filed one: https://github.com/uber/kraken/issues/376